docs/app/pages/components/components-sections/tables/filters/filters.component.ts

```diff
@@ -16,55 +16,55 @@ export class ComponentsFiltersComponent extends BaseDocumentationSection impleme
     filters: Filter[] = [];
 
     statusFilters: Filter[] = [{
-        group: 'status',
```

Comment:
Should these changes also be made in the code sample?

Reply:
It was mainly to prevent the status from being lower case in the documentation.
Without this it will appear as "status" instead of "Status" like it was before the changes. 

Although this could be handled in CSS if you would like?

Reply:
Personally, I don't have a preference, @donna-marijne do you? However my main issue is that the sample code/plunker code eg. snippets/app.ts has not been updated to match, I assume it should be?

Reply:
Just noticed about the snippets folder. I will update these now

Reply:
Great, thanks

Reply:
I think it's fine, as long as the snippets are up to date.

Reply:
Updated the snippets in both UXAspects and ux-aspects-micro-focus